### PR TITLE
Disable embedded server per default

### PR DESCRIPTION
**Pull request by marcust**
As per reasoning in #20

### CODE REVIEW

File: src/main/java/io/vertx/ext/prometheus/VertxPrometheusOptions.java [hunk 1]
```diff
@@ -168,6 +168,16 @@ public boolean isEmbeddedServerEnabled() {
     return this;
   }
 
+  /**
+   * Alias for {@code enableEmbeededServer}
```
Comment (nolequen) [inline src/main/java/io/vertx/ext/prometheus/VertxPrometheusOptions.java]: Please, don't create an alias. Use the existing method or replace it with the new one.

**Comment by nolequen** [comment]
@marcust thanks for your PR.
Could you please fix and squash your commits?

**Review by nolequen — CHANGES_REQUESTED** [review]

**Comment by marcust** [comment]
Done

**Comment by nolequen** [comment]
@marcust thank you!

**Pull request merged**